src/picker/VuePersianDatetimePicker.vue

```diff
@@ -32,6 +32,25 @@
         @focus="focus"
         @blur="setOutput"
       />
+      <arrow
+        v-if="(type === 'date' || type === 'datetime') && dayChangeShortcut"
+        fill="#fff"
+        width="13"
+        direction="right"
+        :style="{ 'background-color': color }"
+        :class="[prefix('daychange'), prefix('nextday')]"
+        @click.native="changeDayBtn(1)"
+      />
+      <arrow
+        v-if="(type === 'date' || type === 'datetime') && dayChangeShortcut"
+        width="13"
+        fill="#fff"
+        direction="left"
+        :style="{ 'background-color': color }"
+        :class="[prefix('daychange'), prefix('previousday')]"
+        @click.native="changeDayBtn(-1)"
+      />
```

Comment:
Please put these two arrow in `template` and apply the condition for the `template`.
for example:
```html
<template v-if="...">
  <arrow  width="13" ... />
  <arrow  width="13" ... />
</template>
```